src/Extension/Choice.php

```diff
@@ -53,5 +53,3 @@ private function buildInternalSequence(&$grammar, $sequence, $grammarParser, $op
         return (count($choice) === 1) ? $choice[0] : $choice;
     }
 }
-
-\ParserGenerator\GrammarParser::$defaultPlugins[] = new Choice();
```

Comment:
All of those lines have been moved to `\ParserGenerator\GrammarParser::addDefaultPlugins`